lib/rules/no-mixed-spaces-and-tabs.js

```diff
@@ -69,6 +66,20 @@ module.exports = function(context) {
         return false;
     }
 
+    /**
+     * Determines if a given line and column are contained in a list of tokens.
+     * @param {array} tokens The array of tokens to look at.
+     * @param {int} line The line to check.
+     * @param {int} column The column to check.
+     * @returns {boolean} True if the line and column are inside any of the tokens.
+     * @private
+     */
+    function isInTokens(tokens, line, column) {
```

Comment:
This is pretty inefficient because it always good through every comment. In reality, you can eliminate more comments as you go through the file. If like to see a more efficient approach here.